tests/vufind.php-cs-fixer.php

```diff
@@ -19,12 +19,21 @@
     'concat_space' => ['spacing' => 'one'],
     'ereg_to_preg' => true,
     'get_class_to_class_keyword' => true,
+    'global_namespace_import' => [
+        'import_constants' => null,
+        'import_functions' => true,
+        'import_classes' => null,
```

Comment:
I see that the default value for import_classes is true. I assume you're setting it to null here to avoid lots of unrelated changes getting mixed together -- which I think is a good idea! But should we consider setting this to true in a follow-up PR?

Reply:
Great, please feel free to try it out after I merge this, and if you like the results, you can open a follow-up PR!